pkg/bmc/client/factory/factory.go

```diff
@@ -0,0 +1,21 @@
+package factory
```

Comment:
This package is redundant. There's no need for a factory because there's only 1 implementation and its OK to couple ourselves to it for now. That is, bmclib.